etl/steps/data/garden/education/2023-07-17/education_lee_lee.py

```diff
@@ -29,7 +28,7 @@
 
 def add_data_for_regions(tb: Table, regions: List[str], ds_regions: Dataset, ds_income_groups: Dataset) -> Table:
```

Comment:
You're not using `regions` argument here. You might want to either remove it and use `REGIONS` or pass it to the function.